index.test.js

```diff
@@ -1,8 +1,10 @@
 import { it, expect, describe } from "vitest";
 import { remark } from "remark";
-import smartypants from "./";
+import remarkMdx from "remark-mdx";
+import remarkSmartypants from "./";
 
-const { process } = remark().use(smartypants);
+const compiler = remark().use(remarkSmartypants);
+const process = compiler.process.bind(compiler);
```

Comment:
What goal are `.bind()` calls achieving?

Reply:
without`.bind()`, there is an error on the last version of `remark`

```
TypeError: Cannot read properties of undefined (reading 'freeze')
```